features/storage/TESTS/kvstore/static_tests/main.cpp

```diff
@@ -67,6 +69,7 @@ static void parse_default_kv()
 //init the blockdevice
 static void kvstore_init()
 {
+#if defined(MBED_CONF_RTOS_PRESENT)
```

Comment:
If init is skipped, then why is not the whole `.cpp` skipped on the compilation phase? Rest of the test cases are not working anyway.